RFC-004-Bitcoin.md

```diff
@@ -0,0 +1,54 @@
+# Bitcoin Definitions
+
+As per [Bitcoin Core](https://github.com/bitcoin/bitcoin/) concensus.
```

Comment:
What does this line mean?

Reply:
What is Bitcoin

Reply:
Should we mention a version? If we only support p2wsh it could make sense to mention we only support everything after version 0.15 (or whenever it was introduced) or what BIPs need to be enabled (e.g. SegWit).